lib/wasm/src/sections_translator.rs

```diff
@@ -17,6 +17,10 @@ use wasmparser;
 use std::str::from_utf8;
 use environ::ModuleEnvironment;
 
+// this is for no_std builds, but has no affect on regular builds
```

Comment:
Here and elsewhere, say "effect" rather than "affect" ("affect" is usually a verb).

Reply:
Yeah, I'm not sure how I made that mistake. :/